.github/workflows/acceptance.yml

```diff
@@ -1226,3 +1226,173 @@ jobs:
         with:
           name: cypress-videos
           path: packages/volto/cypress/videos
+
+  prefix:
+    if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name != github.event.pull_request.base.repo.full_name
+    name: Prefixed site
+    runs-on: ubuntu-latest
+    timeout-minutes: 35
+    strategy:
+      matrix:
+        node-version: [18.x]
```

Comment:
This test better be run with Node 20

Reply:
Done in https://github.com/plone/volto/pull/3592/commits/3938b217555311e266c5f76d142309f12e622f36